apps/web/src/index.css

```
@@ -134,7 +138,8 @@
   /* ─── Slash command menu ─── */
   .slash-menu {
     @apply absolute z-50 bg-white rounded-xl shadow-2xl border border-notion-border
-           w-72 py-1 overflow-hidden;
+           w-72 py-1 overflow-y-auto;
+    max-height: 320px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)